lib/facter/env_temp_variable.rb

```diff
@@ -0,0 +1,3 @@
+Facter.add(:env_temp_variable) do
+  setcode { ENV['TEMP'] }
```

Comment:
We should ensure the value we return uses backslashes. I say that because this fact is included in the cmd.exe argument list, and if it contains forward slashes, e.g. cmd.exe /c C:/windows/temp\do_something, then cmd.exe will barf. Also, I believe TEMP is always defined, based on https://technet.microsoft.com/en-us/library/cc749104%28v=ws.10%29.aspx?f=255&MSPPError=-2147217396